File: Java_Solutions/Project_Euler_Problem_005.java
public class Project_Euler_Problem_005 
{
	public static void main(String args[])
	{
		int num = 1;
		boolean solutionFound = false;
		
		while(!solutionFound)
		{
			int count = 0;
			for(int x = 1; x < 21; x++)
			{
				if(num % x != 0)
				{
					count += 1;
					break;
				}
			}
			if(count == 0)
			{
				System.out.println("Smallest multiple is " + num);
				solutionFound = true;
			}
			num++;
		}
	}
}

File: PHP_Solutions/Project_Euler_Problem_005.php
<?php

$num = 1;
$solutionFound = false;

while(!$solutionFound)
{
	$count = 0;
	for($x = 1; $x < 21; $x++)
	{
		if($num % $x != 0)
		{	
			$count += 1;
			break;
		}
	}
	if($count == 0)
	{
		echo "The smallest multiple is $num"; // This should output 232792560.
		$solutionFound = true;
	}
	$num++;
}

?>
